### PR TITLE
Feature/makerspace faq changes

### LINKED ISSUE #865: Automate Makerspace FAQ page
The Asks:
- The Makerspace FAQ section should be first when someone navigates to /makers/faq
- Several Q&A on the base FAQ need to be omitted or not-displayed via an invisible parameter 

~~Do we make a new dedicated maker-faq.handlebar?~~
We automate this re-ordering upon `/makers` referral which also triggers certain questions to be made invisible?

Makers are a large subset of our traffic and their FAQ is default on the bottom of the current FAQ page made for the PPE requests on Find the Masks

### PR DESCRIPTION
**Pull request by ngottlieb**
resolves #865

### CODE REVIEW

**Comment by linderr** [comment]
Excited for this deployment :) Thanks @ngottlieb !

File: views/faq.handlebars [hunk 1]
```diff
@@ -40,6 +40,7 @@
               Community FAQ.
             </p>
           </dd>
+          {{#unless (eq dataset "makers")}}
```
Comment (linderr) [inline views/faq.handlebars:43]: Very cool.

File: views/faq.handlebars [hunk 2]
```diff
@@ -230,100 +233,16 @@
...
-              <strong>CA Only</strong>: Find the Masks Canada also collaborates with #thePPEDrive’s
-              call for <a href="https://theppedrive.com/3d-printed-visor">3D printers</a>. Join
-              their network to donate 3D-printed face shields.
-            </p>
-          </dd>
-        </dl>
-      </section>
+      {{#unless (eq dataset "makers")}}
+      {{> 'makers_faq' }}
+      {{/unless}}
```
Comment (linderr) [inline views/faq.handlebars:248]: whoa.

**Comment by ngottlieb** [comment]
@linderr I didn't include the last question that's possible on the Google doc because it didn't seem to have a finalized answer...

> I want makerspace items, how do I get them?
Please use our map …….
US-CA Only: If you would like to purchase face shields at-cost, fill out the Shield Net request form here. 
Shield Net only supplies face shields; for other PPE needs, fill out our #findthemasks PPE request form.

File: public/i18n/en.json [hunk 3]
```diff
@@ -245,14 +245,17 @@
...
-  "ftm-faq-independent-makerspace-answer-3": "Shield Net only supplies face shields; for other PPE needs, fill out our #findthemasks <a class=\"add-donation-site-form\">PPE request form</a>.",
+  "ftm-faq-independent-makerspace-answer-1": "If you are an individual marker, we encourage you to use the #FindtheMakers map to connect with a makerspace or maker group in your community. We encourage this collaborative interaction to avoid pinning your home and protect your privacy.",
+  "ftm-faq-independent-makerspace-answer-2": "If you are a makerspace or maker group of any size, join the #FindtheMakers map <a href=\"https://airtable.com/shruH5B27UP3PqKgg\">here</a>.",
   "ftm-faq-independent-makerspace-answer-4": "<strong>CA Only</strong>: Find the Masks Canada also collaborates with #thePPEDrive’s call for <a href=\"https://theppedrive.com/3d-printed-visor\">3D printers</a>. Join their network to donate 3D-printed face shields.",
   "ftm-faq-sites-non-medical-grade": "Why don't all sites allow non-medical grade items?",
```
Comment (linderr) [inline public/i18n/en.json:260]: This was probably not clear enough on the Google Doc but Stef & I used the "xx only" parts in the doc to flag content meant for country specific FAQ content bound for view/partials. This definitely works for now and up to you whether you want to include on this PR or leave it for another issue? Not sure how crazy it is with the new hiding/moving logic - so cool!! Thanks for doing this

Comment (ngottlieb) [reply public/i18n/en.json:260]: I thought about that, but then saw that the text was an exact match for what was already there so figured it wasn't supposed to change, hahah. I will see what I can do.

Comment (ngottlieb) [reply public/i18n/en.json:260]: @linderr updated so CA Only text only displays on CA

**Pull request merged**